pkg/storage/unified/resource/search.go

```diff
@@ -220,11 +227,21 @@ func (s *searchSupport) init(ctx context.Context) error {
 		}
 	}()
 
+	end := time.Now().Unix()
+	if search.IndexMetrics != nil {
+		search.IndexMetrics.IndexCreationTime.WithLabelValues().Observe(float64(end - start))
```

Comment:
🤔 can this be part of the resource store metrics rather than the IndexMetrics?  then we don't need circular dependencies